app/assets/stylesheets/dlme.scss

```diff
@@ -348,3 +348,29 @@ body {
     text-align: center;
   }
 }
+
+// Title truncation for different views
+@mixin truncate_title {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+}
```

Comment:
since this is usually paired with a `line-clamp` rule, what about parametrizing it? i.e.
```scss
@mixin truncate_title($lines) {
  display: -webkit-box;
  -webkit-box-orient: vertical;
  overflow: hidden;
  -webkit-line-clamp: #{$lines};
}
```
so you can do:
```scss
@include truncate-title(4);
```

Reply:
@thatbudakguy thanks, nice idea!